### DISCUSSION TITLE
GitHub Projects iOS Client/App

### DISCUSSION THREAD

**Discussion by XinyiXiang** [UWAppDev/community#16] [Projects]
## Overview
[GitHub Projects (beta)](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects) are a customizable, flexible tool for planning and tracking work on GitHub.

The goal of this club project is to implement a mobile-friendly client side of the projects (beta) board in order to recreate the convenient collaborative workflow on mobile devices, we would focus on making a task management app. 

## Ways of Contributing
- Read and comprehend the subtopics in this [documentation](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects) under GitHub Issues/Projects (beta)/About projects (beta).
- Individuals with design skills may consider embellishing the basic UI.
- Log your progress by composing descriptive commit messages.
- Submit any remaining question(s) to this discussion thread.

## Expectations
- For everyone:
    - Attend most of the regular meetings.
    - Learn more about [GitHub Issues](https://github.com/features/issues).
- For code contribution:
    - Have a basic understanding of Swift by reading https://docs.swift.org/swift-book/GuidedTour/GuidedTour.html (for those with some programming background), or talk to @ApolloZhu about how to get started.
    - Break the project into actionable, specific tasks and assign oneself to the issue(s) aimed at tackling.

## Join Us
- Communicate through #ios discord channel under ***by category***
- Meeting location: hybrid (Zoom link available in #announcement, physical location at Husky Union Building 337).
- Meeting times: Regular Work/Play Hours (**on Saturdays from 4PM - 6PM)**
- Repo: https://github.com/UWAppDev/iOS-GitHub-Board (participating in #hacktoberfest)

**Comment by ApolloZhu** [comment]
## Similar Projects

- Task Board UI https://github.com/alfianlosari/SwiftUI-TrelloClone
